BaseApp/COD50018.TXT

```diff
@@ -0,0 +1,78 @@
+OBJECT Codeunit 50018 DotNet_WordprocessingDocument
+{
+  OBJECT-PROPERTIES
+  {
+    Date=;
+    Time=;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      DotNetWordprocessingDocument@17024400 : DotNet "'DocumentFormat.OpenXml, Version=2.0.5022.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.DocumentFormat.OpenXml.Packaging.WordprocessingDocument";
+      DotNetMemoryStream@17024402 : DotNet "'mscorlib'.System.IO.MemoryStream";
+
+    [External]
+    PROCEDURE Create@17024401(DocumentType@17024401 : 'Document,Template,MacroEnabledDocument,MacroEnabledTemplate';AutoSave@17024403 : Boolean);

```

Comment:
This should be a separate wrapper arround [WordprocessingDocumentType](https://msdn.microsoft.com/en-us/library/office/documentformat.openxml.wordprocessingdocumenttype.aspx).

Reply:
Actually I thought that since it is just a simple enum, there is no need to move to separate wrapper, but yes, it makes sense, so I fixed it with my last commit.